themes/edx.org/lms/static/js/demographics-collection.js

```diff
@@ -0,0 +1,25 @@
+console.log('Demographics javascript file loaded');
+
+$(document).ready(function() {
+    'use strict';
+
+    $('#demographics-dismiss').click(function(event) {
+        event.preventDefault();
+        return $.ajax({
+            url: '/api/demographics/v1/demographics/status/',
+            type: 'PATCH',
+            headers: {
+                'X-CSRFToken': $.cookie('csrftoken')
+            },
+            dataType: 'json',
+            contentType: 'application/json',
+            data: JSON.stringify({
+                call_to_action_dismissed: true
+            }),
+            context: this,
+            success: function() {
+                $('#demographics-banner').hide();
+            }
```

Comment:
Whats the plan if the call fails? do we display an error?

Reply:
If it fails it will remain open, I am not sure there is value here is showing an error. 

@Tj-Tracy Should we hide on error still? I personally am not sure what a user would do with an error in the UI.

Reply:
I think hiding the error is fine. They may be confused as to why the banner didn't go away, but this isn't something that would happen often enough to warrant adding another error in some form to the dashboard.